app/src/androidTest/java/org/mozilla/tv/firefox/ui/BasicNavigationTest.kt

```diff
@@ -28,7 +28,8 @@ class BasicNavigationTest {
 
     /* ktlint-disable no-blank-line-before-rbrace */ // This imposes unreadable grouping.
     @Test
-    @Ignore
+    // Re-enabling because we run tests on BitBar now instead of the emulator
```

Comment:
Nit: I would replace this line with `This test fails on emulators. CI is run on physical devices on BitBar, so this is acceptable.`.  Otherwise I think it's a little confusing out of context (without the @Ignore annotation, readers won't know that it was previously disabled).